corehq/messaging/scheduling/exceptions.py

```diff
@@ -1,3 +1,7 @@
+class ContentException(Exception):
```

Comment:
nit: how about `InvalidEmailAddressException`? Not sure why we have made this generic

Reply:
could do I guess - that would mean it wouldn't get used for anything else in future though

Reply:
58c7d914700

Reply:
Okay. Other classes in the file looked like they were more specific but this was more generic.